K8s/helm/templates/otel-collector-config.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: collector-config
+data:
+  collector-config.yaml: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:4317
+          http:
+            endpoint: 0.0.0.0:4318
+
+    exporters:
+      prometheus:
+        namespace: default
+        endpoint: 0.0.0.0:8889
+      otlp:
+        endpoint: http://jaeger.monitoring.svc.cluster.local:4317
```

Comment:
This is using the grpc port but using http as the protocol, guessing this port needs to be 4138 as shown in the jaeger service.

Reply:
Actually, I take that back. Looks like there are similar port numbers for jaeger and otel collector. I'll have to take another look.